### PR TITLE
Reut/feat/top great movies

### PR DESCRIPTION
**Pull request by ReaGol**

### CODE REVIEW

File: utils/helper.js [hunk 1]
```diff
@@ -5,3 +5,19 @@ export const customSort = (a, b) => {
 	else if (dateA < dateB) return 1;
 	return 0;
 };
+
+export const shuffleMovies = (topMovies) => {
```
Comment (AmeerLalaDeveloper) [inline utils/helper.js:9]: it would be nice if u call this shuffleArray  , because maybe u gonna use this util function  on other array than movies

Comment (ReaGol) [reply utils/helper.js:9]: But it receives topMovies and relies on the topMovies array length... I will have to change the function to use it on something other than movies...

Comment (AmeerLalaDeveloper) [reply utils/helper.js:9]: yes so what u need to do is write a function that shuffle an array that receive  as parameter it shouldnt care about the length of it or what the array include

File: utils/helper.js [hunk 2]
```diff
@@ -5,3 +5,19 @@ export const customSort = (a, b) => {
 	else if (dateA < dateB) return 1;
 	return 0;
 };
+
+export const shuffleArray = (array, shuffledItems) => {
```
Comment (AmeerLalaDeveloper) [inline utils/helper.js:9]: why do we need the suffledItems

**Pull request merged**